### ISSUE TITLE
ConvKB scores interpretation

### ISSUE THREAD

**Issue by lgalarra** [torchkge-team/torchkge#232]
Hi,

I realized the scoring function (method scoring_function) of the ConvKB model returns two values for a given triple. These are the results of a softmax activation that, from what I understand, has two output neurons that represent the probabilities of a triple being true and false. However I do not know in which order these cases are defined in the neural network. Which output neuron corresponds to which case? I could unfortunately not figure it out from reading the code. 

Thanks in advance for your attention and help!

Best,
Luis

**Comment by armand33** [comment]
Hi, sorry for the late reply. This is indeed another issue with this model's implementation. After the softmax layer, one of the two neuron should be chosen as output scoring function and while this is done in the `inference_scoring_function` it is not in the `scoring_function`. I propose a patch in MR #233. Can you confirm this fixes the model's implementation ?
Thanks

**Comment by lgalarra** [comment]
Thanks for the good news! I can now install it with pip.
